packages/slice-machine/lib/models/common/Environment.ts

```diff
@@ -13,17 +12,24 @@ export interface UpdateVersionInfo {
   updateAvailable: boolean;
 }
 
-export default interface Environment {
-  cwd: string;
+export default interface BackendEnvironment {
   userConfig: UserConfig;
-  hasConfigFile: boolean;
   repo?: string;
-  prismicData: PrismicData;
-  chromatic?: Chromatic;
   updateVersionInfo: UpdateVersionInfo;
   mockConfig: any;
   framework: Framework;
   baseUrl: string;
-  hasGeneratedStoriesPath: boolean;
+  cwd: string;
+  prismicData: PrismicData;
   client: DefaultClient | FakeClient;
 }
+
+export interface FrontEndEnvironment {
```

Comment:
Should extend BackendEnv with the specific properties you want :)

Reply:
Or have a common Env type maybe.
But tbh I don't know if it makes sense, but that would definitely makes it clearer.

Reply:
It depends on what's needed in these two, unless they need all the same data, you shouldn't have extra data on the front that is useful only for the server or the other way around so 2 types makes sense if that's the case